src/commands/sync.js

```diff
@@ -21,6 +21,19 @@ module.exports = {
       abbr: 'ignore-hidden'
     },
     {
+      name: 'selectFromFile',
+      boolean: false,
+      default: false,
```

Comment:
Can we set the default here? The `--help` prints out the default, so this was why I got confused:

```
    --selectFromFile, -select-from-file
                          Sync only the list of selected files or directories in the given file. (default: .datdownload) (default: false)
```